molecule-runtime/build.gradle

```diff
@@ -55,6 +55,50 @@ kotlin {
         implementation libs.androidx.core
       }
     }
+
+    darwinMain {
+      dependsOn(commonMain)
+    }
+
+    quartzCoreMain {
+      dependsOn(darwinMain)
+    }
+
+    iosArm64Main {
+      dependsOn(quartzCoreMain)
+    }
+
+    iosSimulatorArm64Main {
+      dependsOn(quartzCoreMain)
+    }
+
+    iosX64Main {
+      dependsOn(quartzCoreMain)
+    }
+
+    tvosArm64Main {
+      dependsOn(quartzCoreMain)
+    }
+
+    tvosSimulatorArm64Main {
+      dependsOn(quartzCoreMain)
+    }
+
+    tvosX64Main {
+      dependsOn(quartzCoreMain)
+    }
+
+    macosMain {
+      dependsOn(darwinMain)
+    }
+
+    macosArm64Main {
+      dependsOn(macosMain)
+    }
+
+    macosX64Main {
+      dependsOn(macosMain)
+    }
```

Comment:
I intend to make this more succinct before it merges. Notice the lack of tests -- this is something that needs to be smoke tested in a sample or integration test.

Reply:
Got a macos test to pass; iOS and tvOS will require a full integration test. 

I'm struggling a bit to make this more succinct since this represents the following hierarchy. 

* darwinMain
  * quartzCoreMain
    * ios*Main (3 in total)
    * tvos*Main(3 in total)
  * macosMain
    * macosArmMain
    * macosX64Main

I could potentially build this as a list of lists and loop to configure, but that doesn't feel much better. Any tips for me?